gm-tester.sh

```diff
@@ -0,0 +1,39 @@
+URL=localhost:26657/block\?height=3
```

Comment:
as a good practice, please add the header to the top of the file:
`sh` 
```suggestion
#!/bin/sh
URL=localhost:26657/block\?height=3
```